gradle.properties

```diff
@@ -16,7 +16,7 @@
 group=org.apache.kafka
 # NOTE: When you change this version number, you should also make sure to update
 # the version numbers in tests/kafkatest/__init__.py and kafka-merge-pr.py.
-version=0.11.0.0-SNAPSHOT
```

Comment:
The merge script version bump is only required for trunk. It's just the default fix version when it can't be inferred from the branch name.